factory/ethElrondBridgeComponents.go

```diff
@@ -550,47 +552,71 @@ func (components *ethElrondBridgeComponents) Start() error {
 func (components *ethElrondBridgeComponents) createAndStartEthToElrondStateMachine() error {
 	log := core.NewLoggerWithIdentifier(logger.GetOrCreate(ethToElrondName), ethToElrondName)
 
-	//TODO add a Start method on the state machine and move the creation of the state machine in the constructor
```

Comment:
the TODO is still relevant: the creation of the state machine and the polling handler can now be moved inside the constructor, thus simplifying the Start method:
can adapt the TODO (for another PR)
or implement the suggestion :)

Valid for the next create and start function